call-home/src/bin/stats/main.rs

```diff
@@ -29,13 +29,17 @@ mod store;
 #[clap(name = utils::package_description!(), version = utils::version_info_str!())]
 struct Cli {
     /// The url for mbus.
-    #[clap(long, default_value = "nats://mayastor-nats:4222")]
+    #[clap(long, default_value = DEFAULT_MBUS_URL)]
     mbus_url: String,
```

Comment:
Should be a Url type?

Reply:
Function message_bus_init accepts a string so  string is the input 
```
/// Initialise the Nats Message Bus.
pub async fn message_bus_init(server: &str) -> impl crate::Bus {
    NatsMessageBus::new(server).await
} 
```